From 460851c15e807d1a2515dfb985423f8a6624f4aa Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 8 Jan 2020 19:14:19 -0800
Subject: [PATCH 3/3] build: Delete libebl from required libraries during link

This is a static library from elfutils, which is not made available when
static archives are disabled, this can result in link failures e.g.

aarch64-yoe-linux-musl/9.2.0/ld: cannot find -lebl
collect2: error: ld returned 1 exit status
Makefile:93: recipe for target 'makedumpfile' failed

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5dce589..c3a4054 100644
--- a/Makefile
+++ b/Makefile
@@ -51,7 +51,7 @@ OBJ_PART=$(patsubst %.c,%.o,$(SRC_PART))
 SRC_ARCH = arch/arm.c arch/arm64.c arch/x86.c arch/x86_64.c arch/ia64.c arch/ppc64.c arch/s390x.c arch/ppc.c arch/sparc64.c
 OBJ_ARCH=$(patsubst %.c,%.o,$(SRC_ARCH))
 
-LIBS = -ldw -lbz2 -lebl -ldl -lelf -lz
+LIBS = -ldw -lbz2 -ldl -lelf -lz
 ifneq ($(LINKTYPE), dynamic)
 LIBS := $(LIBS)
 endif
-- 
2.24.1

